angular-rails-app/lib/tasks/db_connector.rake

```diff
@@ -299,6 +302,8 @@ namespace :db_connector do
       "LBLVIO-LOW PRESSURE BOILER",
       "LL1081-LOCAL LAW 10/81 - ELEVATOR",
       "LL6291-LOCAL LAW 62/91 - BOILERS",
+      "C-CONSTRUCTION",
+      "B-BOILER",
     ]
```

Comment:
Something that was useful for me in the data-import scripts was to pull important lists of data or column names out of the functions and at the top of the file. Those were the interesting bits, the import function was mostly boilerplate. That's not really true here, but it'd be nice to keep these lists of categories as easy to find as possible. You've already got a few vars up at the top, maybe it's worth adding the dob_violation and 311 types up to the top, or at least rename this variable to make it more obvious?